template/en/default/filterexceptions.pl

```diff
@@ -13,6 +13,12 @@
   'pool.id',
 ],
 
+'hook/bug/create/create-form.html.tmpl' => [
+  'pool.id',
+  'bug.pool_id',
```

Comment:
`bug.pool_id` and `bug.pool_order` are no longer used in the template, so these could be removed from here.
Although the filterexceptions.pl is only for Bugzilla's t/008filter.t test, which the extension does not seem to pass anyway.